src/utils/bindings.ts

```diff
@@ -93,13 +95,15 @@ export const availableBindings = async (): Promise<Shell[]> => {
     }
   }
 
-  const powershellConfigPath = path.join(os.homedir(), "Documents", "WindowsPowershell", "Microsoft.PowerShell_profile.ps1");
-  if (!fs.existsSync(powershellConfigPath)) {
-    bindings.push(Shell.Powershell);
-  } else {
-    const powershellConfigContent = fsAsync.readFile(powershellConfigPath, { encoding: "utf-8" });
-    if (!(await powershellConfigContent).includes(powershellScriptCommand())) {
+  if (process.platform == "win32") {
```

Comment:
I think an explicit error should be there if anyone tries to use `powershell` on non-Windows systems.